sdk/communication/communication-messages-rest/samples-dev/SendAudioMessage.ts

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/**
+ * @summary Send a audio message
+ */
+
+import { AzureKeyCredential } from "@azure/core-auth";
+import NotificationClient, { Send202Response } from "@azure-rest/communication-messages";
+// Load the .env file if it exists
+import * as dotenv from "dotenv";
+dotenv.config();
+
+async function main(): Promise<void> {
+    const credential = new AzureKeyCredential(process.env.ACS_ACCESS_KEY || "");
+    const endpoint = process.env.ACS_URL || "";
+    const client = NotificationClient(endpoint, credential);
+    console.log("Sending message...");
+    const  result = await client.path("/messages/notifications:send").post({
+        contentType: "application/json",
+        body: {
+            channelRegistrationId: process.env.CHANNEL_ID || "",
+            to: [process.env.RECIPIENT_PHONE_NUMBER || ""],
+            kind: "audio",
+            mediaUri: "https://sample-videos.com/audio/mp3/wave.mp3"
+        }
+    });
+
+    console.log("Response: " + JSON.stringify(result, null, 2));
+
+    if (result.status === "202") {
```

Comment:
Usually we use the `isUnexpected helper here

```ts
if (isUnexpected(result)) {
   throw....
}

result.body.receipts.forEach( // you will notice how the `as Send202Response` is no longer necessary
```

Same feedback for any other instance where we check status

Reply:
Fixed it as suggested.